File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/EchoVosController.java
package nl.rsdt.japp.jotial.maps.management.controllers;


/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
public class EchoVosController extends VosController {

    public static final String CONTROLLER_ID = "EchoVosController";

    public static final String STORAGE_ID = "STORAGE_VOS_E";

    public static final String BUNDLE_ID = "VOS_E";

    public static final String REQUEST_ID = "REQUEST_VOS_E";

    public EchoVosController() {

    }

    @Override
    public String getTeam() {
        return "e";
    }

    @Override
    public String getId() {
        return CONTROLLER_ID;
    }

    @Override
    public String getStorageId() {
        return STORAGE_ID;
    }

    @Override
    public String getBundleId() {
        return BUNDLE_ID;
    }
}

File: app/src/main/java/nl/rsdt/japp/jotial/maps/management/controllers/EchoVosController.kt
package nl.rsdt.japp.jotial.maps.management.controllers

import nl.rsdt.japp.jotial.maps.wrapper.IJotiMap


/**
 * @author Dingenis Sieger Sinke
 * @version 1.0
 * @since 31-7-2016
 * Description...
 */
class EchoVosController(jotiMap: IJotiMap) : VosController(jotiMap) {

    override val team: String
        get() = "e"

    override val id: String
        get() = CONTROLLER_ID

    override val storageId: String
        get() = STORAGE_ID

    override val bundleId: String
        get() = BUNDLE_ID

    companion object {

        val CONTROLLER_ID = "EchoVosController"

        val STORAGE_ID = "STORAGE_VOS_E"

        val BUNDLE_ID = "VOS_E"

        val REQUEST_ID = "REQUEST_VOS_E"
    }
}

